app/script/entity/message/MemberMessage.js

```diff
@@ -42,6 +62,18 @@ z.entity.MemberMessage = class MemberMessage extends z.entity.SystemMessage {
         .map(userEntity => userEntity);
     });
 
+    this.joinedUserEntities.subscribe(joinedUserEntities => {
```

Comment:
Are they supposed to be sorted alphabetically somehow? This might still be done by the name string.

Reply:
The userEntities are already sorted (https://github.com/wireapp/wire-webapp/blob/dev/app/script/conversation/ConversationRepository.js#L3515)